component-generator/utils.js

```diff
@@ -16,6 +18,25 @@ function validateComponentName(value) {
   return value;
 }
 
+/**
+ * Sends request to the Netify function to inform about generate-component usage.
+ * @param {string} componentName - component name
+ */
+function sendTrackInfo(componentName) {
+  const { BASE_URL, TRACK_ANONYMOUS_ANALYTICS } = process.env;
+  if (TRACK_ANONYMOUS_ANALYTICS) {
+    const url = `${BASE_URL}/.netlify/functions/track-generate-component`;
+    axios.get(url, { params: { componentName } })
```

Comment:
I think we'll want to treat it as a `POST`?

Reply:
Changed to `POST`